src/Generator/Views/Types.Generated.cshtml

```diff
@@ -164,7 +171,7 @@ namespace Elastic.CommonSchema
     /// </remarks>
 </text>
     }
-    public class @FileGenerator.PascalCase(schema.Name) 
+    public @(schema.Name == "log" ? "partial " : "")class @FileGenerator.PascalCase(schema.Name) 
```

Comment:
Should we just generate all as partial? Allows us to hook in in the future if we need to?

Reply:
Yup, makes sense, will follow up with another commit.